tests/languages/elixir/capture_feature.test

```diff
@@ -7,7 +7,12 @@ fun = &(&1 + 1)
 
 [
 	"fun ", ["operator", "="],
-	["capture", "&Math.zero?/1"],
+	["operator", "&"],
+	["module", "Math"],
+	["punctuation", "."],
+	"zero?",
+	["operator", "/"],
+	["number", "1"],

```

Comment:
Shouldn't this be highlighted as a function?

On that note: Wouldn't it be better to just highlight `zero?` as a function instead of `zero?/1`?

Reply:
I think we should highlight `zero?` and `is_function` as functions and `/1` as operator + number.
I'll push a fix for that